common/mb_controller.c

```diff
@@ -86,6 +86,8 @@ int mb_load_controller_config(pid_parameters_t* pid_params){
     int i;
     int count = 0;
 
+    pid_params->FF_term = .8;
```

Comment:
what is our plan for dealing with the feedforward gain? Put it in mb_defs.h file, or pid.cfg, or actually just leave it here?

Reply:
this will change on the next PR when I put this into the config file

Reply:
@levymp  is going to add it to the pid.cfg in the next edit.